mbta.rb

```diff
@@ -1,13 +1,69 @@
 # Your solution here
-class Subway
-  def stops_between_stations(start_line, start_station, end_line, end_station)
-  end
-end
+
   
 # One line, all the stations on that line
-class Line
+class Subway 
+  attr_accessor :red, :green, :orange
+  def initialize()
+    #when initialize subway do initialize the lines whith it
+      @red = Line.new(['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard', 'Porter', 'Davis', 'Alewife'])
+      @green = Line.new(['Government Center', 'Park Street', 'Boylston', 'Arlington', 'Copley', 'Hynes', 'Kenmore'])
+      @orange = Line.new(['North Station', 'Haymarket', 'Park Street', 'State', 'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills'])
+  end
+
+  def stops_between_stations(start_line,  start_station, end_line, end_station)
+
+    #makee these variables accessable from out this methods
+     @start_line = start_line 
+     @end_line = end_line
+    
+     #check value of START line
+        case (@start_line)
+              when "Red" then startt = red
+              when "Green" then startt = green
+              when "Orange" then startt = orange
+           else return p "Not valied START line"
+        end
+    #check value of END line
+        case (@end_line)
+              when "Red" then endd = red
+              when "Green" then endd = green
+              when "Orange" then endd = orange
+          else return p "Not valied END Line"
+        end
```

Comment:
```ruby
   start_line = self.instance_variable_get("@#{start_line}")
   end_line = self.instance_variable_get("@#{end_line}")
```
this will replace the string with the value of the instance variable with the same name